sorters/rolesorter.cpp

```diff
@@ -56,12 +58,13 @@ QPair<QVariant, QVariant> RoleSorter::sourceData(const QModelIndex &sourceLeft,
 
 int RoleSorter::compare(const QModelIndex &sourceLeft, const QModelIndex& sourceRight, const QQmlSortFilterProxyModel& proxyModel) const
 {
-    QPair<QVariant, QVariant> pair = sourceData(sourceLeft, sourceRight, proxyModel);
+
+  QPair<QVariant, QVariant> pair = sourceData(sourceLeft, sourceRight, proxyModel);
```

Comment:
Some weird indentation here; also:
```suggestion
   const auto pair = sourceData(sourceLeft, sourceRight, proxyModel);
```